src/button-dropdown/__tests__/button-dropdown.test.tsx

```diff
@@ -103,6 +103,63 @@ const items: ButtonDropdownProps.Items = [
       });
     });
 
+    (['normal', 'primary'] as Array<ButtonDropdownProps['variant']>).forEach(variant => {
```

Comment:
Should we add a tests to check the disabled reason does NOT apply for all other variants?

Reply:
already added for variant icon: https://github.com/cloudscape-design/components/pull/2409/files/d0255d27b79ba28d5dc022468983f497153b1fa3#diff-fabcab49c7e2458a84ad6bd3367d42ccdcaa5ed56010102255de22b8f35e5dccR146